scripts/zones/Castle_Zvahl_Keep/Zone.lua

```diff
@@ -1,23 +1,55 @@
 -----------------------------------
 -- Zone: Castle_Zvahl_Keep (162)
 -----------------------------------
+local ID = zones[xi.zone.CASTLE_ZVAHL_KEEP]
+-----------------------------------
 ---@type TZone
 local zoneObject = {}
 
+local teleportTable =
+{
+    [1] = { npc = ID.npc.TELE_CENTER, event = 0, }, -- Teleports player to far NE corner
+    [2] = { npc = ID.npc.TELE_NE,     event = 2, }, -- Teleports player to far SE corner
+    [3] = { npc = ID.npc.TELE_SE,     event = 3, }, -- Teleports player to far SW corner (from middle-SE porter)
+    [4] = { npc = ID.npc.TELE_NW,     event = 1, }, -- Teleports player to far SW corner (from middle-NW porter)
```

Comment:
Original value -> 3
`[4] = 3, -- Teleports player to far SW corner (from middle-NW porter)`

Reply:
I recently did a capture that proves these two were incorrectly swapped. What is in this PR is correct. I will link the video in the preexisting link above. My apologies for not including it earlier!